rest/rest-resources/src/main/java/org/apache/brooklyn/rest/BrooklynWebConfig.java

```diff
@@ -83,6 +83,11 @@
     public final static ConfigKey<String> LDAP_OU = ConfigKeys.newStringConfigKey(
             BASE_NAME_SECURITY+".ldap.ou");
 
+    public final static ConfigKey<Boolean> LDAP_FETCH_USER_GROUPS = ConfigKeys.newBooleanConfigKey(
+            BASE_NAME_SECURITY+".ldap.fetch_user_group",
+            "Whether if user groups should be fetch from LDAP server",
```

Comment:
"Whether user groups should be fetched from the LDAP server"

Reply:
tks, I'll change this